### PR TITLE
More documentation fixes

### LINKED ISSUE #74: Documentation fixes
Our docs have a lot of outdated information. Let's fix what we can.

- Installation
- Explanations
- Howtos

https://github.com/epinio/docs/pull/70

### PR DESCRIPTION
**Pull request by juadk**
Fix #74 
Follow up of #70

### CODE REVIEW

File: src/howtos/install_epinio_on_public_cloud.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ If you do not have an existing cluster, follow the [quickstart](https://cloud.go
 
 ## Installation on Public Clouds using Custom Domain
```
Comment (jimmykarily) [inline src/howtos/install_epinio_on_public_cloud.md:46]: I think you can remove this header or at least rename it to `## Installation`

File: src/howtos/install_epinio_on_rancher_desktop.md [hunk 2]
```diff
@@ -37,6 +40,7 @@ The Epinio installation is pretty much identical on Linux, MacOS and Windows:
 
 2. Download the [latest version](https://github.com/epinio/epinio/releases) of Epinio binary and copy it on your client. It is recommended to add it to the `PATH` variable (`export PATH=<epinio-binary-path>:$PATH` on Linux/MacOS with Bash and on Windows please follow [Kevin Berg's article](https://medium.com/@kevinmarkvi/how-to-add-executables-to-your-path-in-windows-5ffa4ce61a53)).
```
Comment (jimmykarily) [inline src/howtos/install_epinio_on_rancher_desktop.md:41]: Better link to this page? https://docs.epinio.io/installation/install_epinio_cli.html

**Review by jimmykarily — APPROVED** [review]

File: src/howtos/install_epinio_on_rancher_desktop.md (hunk 2 above)
Comment (juadk) [reply src/howtos/install_epinio_on_rancher_desktop.md]: Done, but this step is (will be) optional to me, especially with the standalone UI in.

File: src/howtos/install_epinio_on_k3d.md [hunk 3]
```diff
@@ -25,17 +28,14 @@ After the command returns, `kubectl` should already be talking to your new clust
...
 $ kubectl get nodes
 NAME                  STATUS   ROLES                  AGE   VERSION
-k3d-epinio-server-0   Ready    control-plane,master   38s   v1.20.0+k3s2
+k3d-epinio-server-0   Ready    control-plane,master   38s   v1.22.6+k3s1
 ```
 
 ### Install Epinio on the Cluster
 
-Follow ["magic" DNS setup](../installation/magicDNS_setup.md) to install Epinio in your test environment.
+Follow ["magic" DNS setup](../installation/magicDNS_setup.md) to define the system domain you will use for Epinio.
```
Comment (nunix) [inline src/howtos/install_epinio_on_k3d.md:36]:
```suggestion
Follow ["magic" DNS setup](../installation/magicDNS_setup.md) to define the domain name you will use for Epinio.
```

File: src/howtos/install_epinio_on_k3s.md [hunk 4]
```diff
@@ -1,13 +1,18 @@
...
+* k3s version v1.23.4+k3s1 and v1.22.7+k3s1 
+* openSUSE Leap 15.3 and Tumbleweed
 
 ## Get K3s Kubernetes Cluster
 
 ### Install K3s
 
 Follow the [instructions](https://k3s.io/) to install k3s on your system.
 
+Default options given by the k3s versions mentioned just above were used.
```
Comment (nunix) [inline src/howtos/install_epinio_on_k3s.md:14]:
```suggestion
The K3s versions used in this how-to are the ones mentioned just above, with all the options set by default.
```

File: src/howtos/install_epinio_on_k3s.md [hunk 5]
```diff
@@ -1,13 +1,18 @@
...
+This page was written using the following versions:
+* [epinio helm chart 0.7.1](https://github.com/epinio/helm-charts/releases/tag/epinio-0.7.1)
+* k3s version v1.23.4+k3s1 and v1.22.7+k3s1 
+* openSUSE Leap 15.3 and Tumbleweed
 
 ## Get K3s Kubernetes Cluster
 
 ### Install K3s
 
 Follow the [instructions](https://k3s.io/) to install k3s on your system.
```
Comment (nunix) [inline src/howtos/install_epinio_on_k3s.md:12]:
```suggestion
Follow the [instructions](https://k3s.io/) to install K3s on your system.
```

File: src/howtos/install_epinio_on_minikube.md [hunk 6]
```diff
@@ -47,4 +50,5 @@ Now you can install Epinio on the Cluster.
 
 ### Install Epinio on the Cluster
 
-Follow ["magic" DNS setup](../installation/magicDNS_setup.md) to install Epinio in your test environment.
+With Minikube, you will need to deploy Traefik and cert-manager.
```
Comment (nunix) [inline src/howtos/install_epinio_on_minikube.md:53]:
```suggestion
With Minikube, you'll need to deploy Traefik and cert-manager.
```

File: src/howtos/install_epinio_on_k3d.md [hunk 7]
```diff
@@ -1,5 +1,8 @@
 # Creating a K3d Kubernetes Cluster
 
+This page was written using the following versions:
```
Comment (nunix) [inline src/howtos/install_epinio_on_k3d.md:3]:
```suggestion
This how-to was written using the following versions:
```

File: src/howtos/install_epinio_on_k3s.md [hunk 8]
```diff
@@ -1,13 +1,18 @@
 # Creating a K3s Kubernetes Cluster
 
-Note: this documentation was tested with Epinio v0.1.3 and k3s v1.21.5+k3s2 on openSUSE Leap 15.3.
+This page was written using the following versions:
```
Comment (nunix) [inline src/howtos/install_epinio_on_k3s.md:3]:
```suggestion
This how-to was written using the following versions:
```

File: src/howtos/install_epinio_on_minikube.md [hunk 9]
```diff
@@ -1,5 +1,8 @@
 # Creating a Minikube Kubernetes Cluster
 
+This page was written using the following versions:
```
Comment (nunix) [inline src/howtos/install_epinio_on_minikube.md:3]:
```suggestion
This how-to was written using the following versions:
```

File: src/howtos/install_epinio_on_rancher.md [hunk 10]
```diff
@@ -4,28 +4,53 @@ Before installing Epinio, you need a running Rancher instance.
 
 If you do not have such, please use the Rancher's [installation guides](https://rancher.com/docs) to set one up.
 
-When I wrote this documentation, I used the following versions:
-* epinio-installer (Helm chart) 0.3.5 
+This page was written using the following versions:
```
Comment (nunix) [inline src/howtos/install_epinio_on_rancher.md:7]:
```suggestion
This how-to was written using the following versions:
```

File: src/howtos/install_epinio_on_rancher.md [hunk 11]
```diff
@@ -4,28 +4,53 @@ Before installing Epinio, you need a running Rancher instance.
...
 Go to `Global settings` > `Advanced settings`  and update the following settings:</br>
 
 | | |
 |--|--|
-| `ui-dashboard-index`  | `https://releases.rancher.com/dashboard/epinio-v0.3.0/index.html` |
+| `ui-dashboard-index`  | `https://releases.rancher.com/dashboard/epinio-v0.6.0/index.html` |
 | `ui-offline-preferred`  | `Remote` |
 
-*Please, keep in mind that `ui-dashboard-index` value will change over the time until it is not merged in stable Rancher version.*
+*Please, keep in mind that `ui-dashboard-index` value will change over the time until it is merged in stable Rancher version.
```
Comment (nunix) [inline src/howtos/install_epinio_on_rancher.md:27]:
```suggestion
> NOTE: Please keep in mind that `ui-dashboard-index` value will change over time until it's merged in stable Rancher version.
```

File: src/howtos/install_epinio_on_rancher.md [hunk 12]
```diff
@@ -4,28 +4,53 @@ Before installing Epinio, you need a running Rancher instance.
...
+
+## Epinio Prerequisites
+
+### Install an Ingress controller
+
+If there is no Ingress controller running yet on the cluster, install Traefik.
```
Comment (nunix) [inline src/howtos/install_epinio_on_rancher.md:33]:
```suggestion
If there's no Ingress controller running yet on the cluster, you need to install one. In the example below, Traefik will be installed.
```

Comment (nunix) [reply src/howtos/install_epinio_on_rancher.md:33]: The Traefik installation link might need to be updated/standardized if we explain it somewhere else.

File: src/howtos/install_epinio_on_rancher.md [hunk 13]
```diff
@@ -4,28 +4,53 @@ Before installing Epinio, you need a running Rancher instance.
...
+Go to `Apps & Marketplace` >  `Repositories` > `Create`.
+
+Give a name to the repo and use `https://helm.traefik.io/traefik` as the index URL.
+
+Once done, go to the "Charts" menu and install the `traefik` chart.
+See also the [Install Epinio](../installation/installation.html#ingress-controller) page for some helm values you can set.
+
+### Install cert-manager
+
+If cert manager is not installed yet you can install it by adding the relevant repository
```
Comment (nunix) [inline src/howtos/install_epinio_on_rancher.md:47]:
```suggestion
If cert manager is not installed yet, you can install it by adding the relevant repository
```

File: src/howtos/install_epinio_on_rancher.md [hunk 14]
```diff
@@ -35,26 +60,27 @@ Add the Epinio Helm repo in `Apps & Marketplace` >  `Repositories` > `Create`
 
 Give a name to the repo and use `https://epinio.github.io/helm-charts` as index URL.
 
-Once done, go to the Chart menu and install the `epinio-installer` chart.
+Once done, go to the "Charts" menu and install the `epinio` chart.
 
-While you can select the version you want most of the time you should choose the latest one (default). Then click install in the top right corner.
+While you can select the version you want most of the time you should choose the latest one (default). This page was written with version `0.7.1`. Then click install in the top right corner.
```
Comment (nunix) [inline src/howtos/install_epinio_on_rancher.md:65]:
```suggestion
> NOTE: While you can select the version you want, it's strongly advised you choose the latest one (default). This how-to was written with version `0.7.1`. 
```

File: src/howtos/install_epinio_on_rancher.md [hunk 15]
```diff
@@ -35,26 +60,27 @@ Add the Epinio Helm repo in `Apps & Marketplace` >  `Repositories` > `Create`
...
 
 On the next screen, you can select a namespace where the deployment will happen. You also have to set a name for the deployment.
```
Comment (nunix) [inline src/howtos/install_epinio_on_rancher.md:67]:
```suggestion
Click install in the top right corner.

On the next screen, you can select a namespace where the deployment will happen. You also have to set a name for the deployment.
```

File: src/howtos/install_epinio_on_rancher.md [hunk 16]
```diff
@@ -35,26 +60,27 @@ Add the Epinio Helm repo in `Apps & Marketplace` >  `Repositories` > `Create`
...
+- `Domain`: [Explained here](../installation/dns_setup.md)
+- `Access control allow origin`:  It is mandatory to insert your Rancher dashboard URL into this field. It basically allows Rancher to talk with the Epinio API.
 
 Once everything is configured, click on the install button and then wait a few minutes for the installation to complete.</br>
 After that follow the instructions given by the Helm install command output.
 
 ## Access the Epinio menu
 
-Click on the Epinio icon in the main side menu (under **GLOBAL APPS**).
+Click on the "burger" menu icon at the top left and then the Epinio icon under **GLOBAL APPS**.
```
Comment (nunix) [inline src/howtos/install_epinio_on_rancher.md:79]:
```suggestion
Click on the "burger" menu icon (☰) at the top left and then the Epinio icon under **GLOBAL APPS**.
```

File: src/howtos/install_epinio_on_rancher.md [hunk 17]
```diff
@@ -35,26 +60,27 @@ Add the Epinio Helm repo in `Apps & Marketplace` >  `Repositories` > `Create`
...
 
 You should see your fresh Epinio instance there.
 
 __Attention__. If you use an untrusted certificate, you have to click on the URL, accept the security exception and use the refresh button shown below the page title, i.e. __Epinio instances__.</br>
```
Comment (nunix) [inline src/howtos/install_epinio_on_rancher.md:83]:
```suggestion
> ATTENTION: If you use an untrusted certificate, you have to click on the URL, accept the security exception and use the refresh button shown below the page title, i.e. __Epinio instances__.
```

File: src/howtos/install_epinio_on_rancher_desktop.md [hunk 18]
```diff
@@ -1,5 +1,8 @@
 # Rancher Desktop configuration
 
+This page was written using the following versions:
```
Comment (nunix) [inline src/howtos/install_epinio_on_rancher_desktop.md:3]:
```suggestion
This how-to was written using the following versions:
```

File: src/howtos/install_epinio_on_rancher_desktop.md [hunk 19]
```diff
@@ -12,7 +15,7 @@ Install the [latest version](https://github.com/rancher-sandbox/rancher-desktop/
 
 When running Rancher Desktop for the first time wait until the initialization is completed.
```
Comment (nunix) [inline src/howtos/install_epinio_on_rancher_desktop.md:16]:
```suggestion
When running Rancher Desktop for the first time, wait until the initialization is completed.
```

File: src/howtos/install_epinio_on_rancher_desktop.md [hunk 20]
```diff
@@ -35,8 +38,9 @@ If you don't want to use the default "magic" DNS, please follow [DNS setup](dns_
...
 
-2. Download the [latest version](https://github.com/epinio/epinio/releases) of Epinio binary and copy it on your client. It is recommended to add it to the `PATH` variable (`export PATH=<epinio-binary-path>:$PATH` on Linux/MacOS with Bash and on Windows please follow [Kevin Berg's article](https://medium.com/@kevinmarkvi/how-to-add-executables-to-your-path-in-windows-5ffa4ce61a53)).
+2. Install the [Epinio CLI](../installation/install_epinio_cli.md).
 
-3. Use `helm` to install Epinio by following either the [Method 1](install_epinio_auto.md) (recommended for most cases) or the [Method 2](install_epinio_manual.md) (useful for advanced configuration or if you already have some components in your environment). As **Traefik** is already installed, just bypass this step (by adding option `--set skipTraefik=true` for Method 1).
+3. Follow the [Epinio installation process](../installation/installation.md).
 
 **NOTE:** *there is currently a [blocking issue](https://github.com/rancher-sandbox/rancher-desktop/issues/576) on Linux which prevent Epinio to push application!*
+*However, you will find a workaround at the end of the issue.*
```
Comment (nunix) [inline src/howtos/install_epinio_on_rancher_desktop.md:46]:
```suggestion
> NOTE: there is currently a [blocking issue](https://github.com/rancher-sandbox/rancher-desktop/issues/576) on Linux which prevent Epinio to push application!
> However, you will find a workaround at the end of the issue.
```

**Review by nunix — COMMENTED** [review]
Minor rephrasings

File: src/howtos/install_epinio_on_rancher.md (hunk 12 above)
Comment (juadk) [reply src/howtos/install_epinio_on_rancher.md]: In that case, I think I can put this link: https://docs.epinio.io/installation/installation.html#ingress-controller

**Review by nunix — APPROVED** [review]

**Pull request merged**